### PR TITLE
[KO] Add mappers' guild translation

### PR DESCRIPTION
**Pull request by hikizisa**

### CODE REVIEW

File: wiki/Mappers_Guild/ko.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+tags:
+  - Mapper's Guild
+  - MG
+---
+
+# 매퍼스 길드
+
+**매퍼스 길드**는 osu!의 [피쳐드 아티스트](/wiki/Featured_Artists)를 위한 [비트맵](/wiki/Beatmaps)을 만드는 책임이 있는 매퍼들의 그룹입니다. MMO-스타일의 길드 형식을 차용해서, 이는 매퍼들이 피쳐드 아티스트 비트맵의 랭크 과정에 참여하고, 매핑과 관련된 퀘스트를 위해 협력하면서 보상을 받을 수 있도록 합니다.
+
+길드의 조직 구성과 토론은 [osu!개발 디스코드 서버](https://discord.gg/ppy)의 `#mappers-guild` 채널에서 찾을 수 있습니다.
```
Comment (inix1257) [inline wiki/Mappers_Guild/ko.md:11]: osu! 개발 <- 띄어쓰기

**Review by inix1257 — DISMISSED** [review]

File: wiki/Mappers_Guild/ko.md (hunk 1 above)
Comment (hikizisa) [reply wiki/Mappers_Guild/ko.md]: 수정했습니다!
(not actually dismissed, fixed)

**Comment by inix1257** [comment]
seems alright

**Comment by hikizisa** [comment]
finished rebasing now

**Comment by Garalulu** [comment]
Good to go

**Review by TPGPL — APPROVED** [review]

**Pull request merged**